extensions/messages/src/send-message.tsx

```diff
@@ -67,6 +83,10 @@ export default function Command({
       if (result === "Success") {
         const name = getName(correspondingContact);
 
+        if (shouldCloseMainWindow) {
+          await showHUD(`Sent Message to ${name}`, { clearRootSearch: true });
+        }
```

Comment:
You can actually call `closeMainWindow` here instead of calling `showHUD`, the toast will still be shown as a HUD.